cmd/maya-exporter/app/command/exporter_test.go

```diff
@@ -77,3 +81,38 @@ func startTestServer(t *testing.T, options *VolumeExporterOptions, errMsg chan e
 		errMsg <- options.StartMayaExporter()
 	}()
 }
+
+func TestJsonHandler(t *testing.T) {
+	cases := map[string]struct {
+		targetURL string
+		err       error
+	}{
+		"When URL is correct": {
+			targetURL: "http://localhost:9500" + metricsPath + "json/",
+			err:       nil,
+		},
+	}
+
+	srv := &http.Server{Addr: ":9500"}
+	http.HandleFunc(metricsPath+"json/", jsonHandler)
+	go func() {
+		if err := srv.ListenAndServe(); err != nil {
```

Comment:
can we use test http server instead of this ?
Please have a look at https://blog.questionable.services/article/testing-http-handlers-go/

Reply:
sure